src/ckb/keywidget.cpp

```diff
@@ -311,6 +311,12 @@ void KeyWidget::paintEvent(QPaintEvent*){
             float y = key.y + offY - 1.8f;
             float w = 3.6f;
             float h = 3.6f;
+            /*if(model == KeyMap::K55){
```

Comment:
Ideally, the drawn colour zones in the GUI should be much bigger. I should look into this one more time before merging, see what can be done, and if not, remove this block.